otelcollector/telegraf/telegraf-prometheus-collector.conf

```diff
@@ -27,6 +27,7 @@
   mode = "$MODE"
   winmode = "$WINMODE"
   macmode = "$MAC"
+  opt = "true"
```

Comment:
Thanks for fixing this Vishwa! Is this dim just to be able to query for clusters that have this new conf rolled out?